go/enclave/enclave.go

```diff
@@ -251,6 +251,11 @@ func NewEnclave(
 		logger.Crit("failed to resync L2 chain state DB after restart", log.ErrKey, err)
 	}
 
+	err = scb.Load()
+	if err != nil && !errors.Is(err, errutil.ErrNotFound) {
```

Comment:
do you handle not-found in any way?

Reply:
No, not found means db is there, but batches are not, which in turn means we havent bootstrapped yet with the first batches. Unless we delete them, but we should avoid pruning the first two.